src/go.mod

```diff
@@ -4,6 +4,8 @@ go 1.23.0
 
 toolchain go1.23.5
 
+replace github.com/spf13/cobra v1.8.0 => github.com/DefangLabs/cobra v0.1.0
```

Comment:
We should follow their versioning, whichever version we ended up forking, and append a label at the end, eg. `v1.8.0-defang` or so.